Signal/src/ViewControllers/AddContactShareToExistingContactViewController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ContactsUI
+
+class AddContactShareToExistingContactViewController: ContactsPicker, ContactsPickerDelegate, CNContactViewControllerDelegate {
```

Comment:
Rather than subclass the ContactsPicker, I would have preferred to compose it, using the contactsPickerDelegate, but it seemed like that was going to require some bigger changes to the contactsShareViewHelper.